Makefile

```diff
@@ -48,6 +48,8 @@ build: prebuild
 	@$(MAKE) build/deb/$(NAME)_$(VERSION)_amd64.deb
 	@$(MAKE) build/deb/$(NAME)_$(VERSION)_arm64.deb
 
+build-native: build/$(SYSTEM_NAME)/$(NAME)-$(HARDWARE)
+
```

Comment:
thought: We can omit this change, I just found it very handy to be able to run `make build-native` on both my mac and Linux machine 🙂